README.md

```diff
@@ -3,17 +3,18 @@
 # Portfolio - Mobile skeleton app
 
 > Module 1 portfolio project, skeleton app page with no functionality.
+> The mobile page should be a clone of [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)
 
-<!--![screenshot](./app_screenshot.png)-->
+Here you can see a picture ---> [screenshot](./app_screenshot.png)
```

Comment:
- [ ] Please fix this screenshot here. You can always take a look back at the README [template](https://github.com/microverseinc/readme-template) to remind yourself
- [ ] Also, kindly fix the broken MIT link. I couldn't comment around the line downside because it's not from the changed files.
Please keep in mind that such a very small mistake could leave a bad impression on future reviewers.